src/graphql/node/input.rs

```diff
@@ -138,27 +138,50 @@ pub(super) fn create_draft_update(
         .map(|agent| (agent.key.clone(), agent.config.clone()))
         .collect();
 
-    let agents: Vec<review_database::Agent> = if let Some(new_agents) = new.agents {
-        new_agents
-            .into_iter()
-            .map(|agent_draft| {
-                let config = old_config_map
-                    .get(&agent_draft.key)
-                    .and_then(|config| config.as_ref().and_then(|c| c.clone().try_into().ok()));
-
-                review_database::Agent {
-                    node: u32::MAX,
-                    key: agent_draft.key,
-                    kind: agent_draft.kind.into(),
-                    status: agent_draft.status.into(),
-                    config,
-                    draft: agent_draft.draft.and_then(|draft| draft.try_into().ok()),
-                }
+    let agents: Vec<review_database::Agent> =
+        new.agents
+            .map(|new_agents| {
+                new_agents
+                    .into_iter()
+                    .map(|agent_draft| {
```

Comment:
I think `agent_draft` can bring misunderstanding in this context, because it looks like the draft configuration of the, but it is actually not. I think we can use `new_agent` instead.